src/content/docs/blog/biome-v2.mdx

```diff
@@ -0,0 +1,266 @@
+---
+title: Biome v2 - Type-aware linter
+excerpt: |
+  Biome 2.0 is officially out, as the first JavaScript linter that provides type-aware linting rules
+  that don't rely on the TypeScript compiler. With this release, Biome also ships plugin support, multi-file analysis,
+  revamped import sorting, experimental HTML formatting, better suppressions, better LSP support, and more!
+authors:
+  - ema
+  - team
+date: 2025-06-02
+cover:
+    light: "@/assets/blog/roadmap-2024/banner-light.png"
+    dark: "@/assets/blog/roadmap-2024/banner-dark.png"
+    alt: The brand of the project. It says "Biome, toolchain of the web"
+socialImage: "@/assets/social-logo.png"
+featured: true
+---
+
+import { Image } from "astro:assets"
+import darkVercel from "../../../assets/sponsors/vercel-logotype-dark.png";
+import lightVercel from "../../../assets/sponsors/vercel-logotype-light.png";
+import darkDepot from "../../../assets/sponsors/depot-logo-horizontal-on-dark@3x.png";
+import lightDepot from "../../../assets/sponsors/depot-logo-horizontal-on-light@3x.png";
+import Avatar from "@/components/Avatar.astro";
+
+We are happy to announce that Biome v2 is officially out! 🍾 Biome v2 is the _first_ JavaScript and TypeScript linter that provides
+**type-aware linting rules that doesn't rely on the TypeScript compiler**! This means that you can lint your project
+without necessarily installing the `typescript` package.
+
+With this release, the [Core Contributors of the project](/internals/credits#core-contributors) want to show
+to the whole community and web ecosystem that Biome is here to stay and deserves to earn its place as the next-generation toolchain for the web.
+No other tools have achieved this great milestone in such a short amount of time ([two years](/blog/annoucing-biome)) and resources. This has been possible
+thanks to the companies and people who believed in the project.
+
```

Comment:
May be good to add one more paragraph:

```suggestion

Preliminary testing shows that our [`noFloatingPromises` rule](/linter/rules/no-floating-promises/), which is based on our new type inference work, can detect floating promises in about 60% of the cases that would be detected by using `typescript-eslint`, at a fraction of the performance impact. And needless to say, we have plenty of ideas on how to improve this metric even further.

Keep in mind that your mileage may vary, as these early numbers are based on a limited set of use cases. Nevertheless, we look forward to people trying it out and reporting their experiences so that we can quickly reach a level of confidence that would be sufficient for most projects.
```